src/Go/boardAnalysis/goAI.ts

```diff
@@ -27,7 +27,7 @@ type PlayerPromise = {
   resolver: ((play?: Play) => void) | null;
 };
 
-const gameOver = { type: GoPlayType.gameOver, x: null, y: null } as const;
+const gameOver: Play = { type: GoPlayType.gameOver, x: null, y: null } as const;
```

Comment:
The type here is unneeded. TS is structurally typed, not declaratively typed, so this is only telling the compiler what it already knows.

Reply:
I thought so, but this made Idea stop complaining about it 🤷‍♂️